src/test/scala/dsptools/DspContextSpec.scala

```diff
@@ -7,6 +7,11 @@ import dsptools.numbers._
 import org.scalatest.freespec.AnyFreeSpec
 import org.scalatest.matchers.should.Matchers
 
+// The following line is necessary for running the parallel test section but is not compatible
+// with 2.12.
+//import scala.collection.parallel.CollectionConverters.RangeIsParallelizable
+//
```

Comment:
Please don't just add commented code. If you really need this import, then you can add a file with a shim object that doesn't do anything, see `SeqMap` in chisel3 which is provided as an alias to `ListMap` for Scala 2.12: https://github.com/chipsalliance/chisel3/blob/master/core/src/main/scala-2.12/scala/collection/immutable/package.scala

I'd just add the following to a file `src/main/scala-2.12/scala/collection/parallel/CollectionConverters.scala`:
```scala
package scala.collection.parallel

object CollectionConverters {
  object RangeIsParallelizable
}
```